H/hidapi/build_tarballs.jl

```diff
@@ -48,6 +48,6 @@ dependencies = [
     Dependency("libusb_jll")
 ]
 
-platforms = [p for p in supported_platforms() if !Sys.isfreebsd(p)]
+platforms = [p for p in supported_platforms(;experimental=true) if !Sys.isfreebsd(p)]
```

Comment:
This can be simplified a bit with
```suggestion
platforms = supported_platforms(;exclude=Sys.isfreebsd)
```